### ISSUE TITLE
I2c exceptions?

### ISSUE THREAD

**Issue by dougthebug** [dsmithson/SpyderTallyController#11]
Hi Derek, I'm running your pi image without the LCD or the relays for basic testing and to ensure the pi I have works...  Do I have to have the LCD wired up for the server to run?  I'm happy to post my log but I'm starting to assume I need the I2c bus going to make this run.

Thanks and Happy New Year!

**Comment by dsmithson** [comment]
Hello!  Yeah you'll probably need to go into raspi-config and enable I2C to prevent errors from popping up with the LCD display, even if you aren't physically using one.  

Good catch and thanks for the note.  I should probably put some handling in place to just quietly ignore front panel init issues when I2C fails (or log it during startup but not crash), and maybe even add something to the deviceConfig.json to allow enabling/disabling the front panel entirely.

**Comment by dougthebug** [comment]
Thanks Derek,

I’ve got some 16x2s lying around somewhere, and I’ll look into the raspi-config settings, and get back to you.

**Comment by craigreilly** [comment]
@dougthebug Did you decipher how to stop this error without a display? I looked in the config.txt file and could not find what to enable or disable...

**Comment by dsmithson** [comment]
Thanks for the nudge @craigreilly  - I pulled my Spyder Tally off the shelf and was able to recreate this pretty easily.  Initially when @dougthebug reported it I figured he just needed to enable I2C in raspi-config, but I was wrong and indeed the display init will crash just from not being present.

Latest code I just pushed up will catch and log display initialization failure, and the app will continue happily working.
